### DISCUSSION TITLE
Get the geometry of the selected shape

### DISCUSSION THREAD

**Discussion by giswqs** [geoman-io/leaflet-geoman#1542] [Questions & Help]
How to get the geometry of the selected shape? I am trying to use the ipyleaflet[ Geoman Draw Control](https://ipyleaflet.readthedocs.io/en/latest/controls/geoman_draw_control.html). I can't see to find a way to get the geometry of the selected shape. 

![Peek 2024-11-30 22-54](https://github.com/user-attachments/assets/59ea0c6f-a4e9-43be-bc39-8fa1009b6814)

**Comment by Falke-Design** [comment]
Hi,
we are vanilla Leaflet-Geoman and we can provide support for other frameworks / tools like ipyleaflet.

I suggest you to search in the ipyleaflet / leafler documentation for a click listener for the layer.

Like
```
layer.on('click', (e)=>console.log(e));
```
